example.json

```diff
@@ -0,0 +1,21 @@
+{
+    "_id": "c-35",
+    "_parentId":"b-20",
```

Comment:
needs a space between the colon and the opening quote (also in `_type` and `instruction`)